use_case_examples/hybrid_model/serve.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
same comment as for the compile.sh file, why not sure argparse directly into serve_model.py ?

Reply:
@fd0r if you removed compile.sh then maybe you could do the same for serve.py ?